src/lib/containers/home.tsx

```diff
@@ -158,11 +158,23 @@ export default Relay.createContainer(Home, {
 interface RelayProps {
   home: {
     hero_units: Array<any | null> | null,
-    artwork_modules: Array<{
-      __id: string,
-    } | null> | null,
-    artist_modules: Array<{
-      __id: string,
-    } | null> | null,
-  },
+    artwork_modules:
+      | Array<
+
```

Comment:
https://github.com/prettier/prettier/commit/c8f255d6d574054e170692f3bc09683334a011cb Fixes it